apps/src/code-studio/components/progress/StageProgress.story.jsx

```diff
@@ -29,7 +28,6 @@ export default storybook => {
           url: "http://studio.code.org/s/csp1/stage/2/puzzle/1",
           freePlay: false,
           progression: "Lesson Vocabulary & Resources",
-          status: LevelStatus.not_tried
```

Comment:
These are not actually needed. In both the new and old components, we should be getting status from elsewhere in the redux store.